rhcos-tools.spec

```diff
@@ -0,0 +1,37 @@
+Name:           rhcos-tools
+Version:        0.0.0
+Release:        1%{?dist}
```

Comment:
Minor point: we are moving RHCOS packages to releases like: `1.${branch}.%{?dist}`, perhaps we should consider that when we do branches for this package.

Reply:
Updated